lib/ValidateDirectiveVisitor.ts

```diff
@@ -200,7 +204,7 @@ const checkMustValidateInputField = (
   return checkMustValidateInput(field.type);
 };
 
-const getFinalType = (
+export const getFinalType = (
```

Comment:
i don't think the export is needed (at least I couldn't find it being imported)

Reply:
Ops, I exported it while trying another approach to the update and forgot to remove the export after dropping it.